src/css/responsive/tablet.styl

```diff
@@ -60,6 +60,33 @@ svg.pagination-arrow
 
 // ******************************************************
 
+// BLOG
+
+.blog-item-single
+  max-width: none
+
+body#about
```

Comment:
there is no need to tag qualify an ID selector. Makes the selector less efficient: https://css-tricks.com/efficiently-rendering-css/#article-header-id-2